src/features/base-layout/components/header/header.tsx

```diff
@@ -27,10 +42,8 @@ export const Header = () => {
   useEffect(() => {
     const listenScrollEvent = () => {
       const scrollY = window.scrollY;
-      if (scrollY < 65) {
+      if (scrollY < 500) {
```

Comment:
Please consider adding a comment, because I have no idea what this does

Reply:
Added a comment to the code
Corrected in [refactor: changes in accordance with SpaNb4 comments](https://github.com/rolling-scopes/site/pull/258/commits/14a9752b951595f0aded14a005129e1f883aa9cf)